js/tests/views/settings_spec.js

```diff
@@ -36,14 +36,6 @@ define(['views/settings', 'views/helper'], function(SettingsView) {
 		it('produces the correct HTML', function () {
 			settingsview.render();
 
-			html = settingsview.el.innerHTML.trim();
```

Comment:
:) I know. HTML comparison is a pain. Lets check for the link text only. Should be good enough? Or?